packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -68,6 +68,70 @@ function coverInitialTxValue(tx) {
   return tx
 }
 
+function resolveArgsForSignTransaction(args) {
+  if (args.length === 0 || args.length > 3) throw new Error('Invalid parameter: The number of parameters is invalid.')
+  
+  // privateKey and callback are optional parameter
+  // "args.length === 2" means that user sent parameter privateKey or callback
+  let tx = args[0], privateKey, callback
+
+  if (!tx || (!_.isObject(tx) && !_.isString(tx))) {
+    throw new Error('Invalid parameter: The transaction must be defined as an object or RLP encoded string')
+  }
+
+  if (args.length === 2) {
+    if (_.isFunction(args[1])) {
+      callback = args[1]
+    } else {
+      privateKey = args[1]
+    }
+  } else if (args.length === 3) {
+    if (args[1] && typeof args[1] !== 'string' && !_.isArray(args[1])){
+      return handleError('Invalid parameter: The parameter for the private key is invalid')
+    }
+    privateKey = args[1]
+    callback = args[2]
+  }
+
+  // For handling when callback is undefined.
+  callback = callback || function () {}
+
+  return { tx, privateKey, callback }
+}
+
+function resolveArgsForFeePayerSignTransaction(args) {
```

Comment:
Please add description of this function.

Reply:
Description for resolveArgsForFeePayerSignTransaction is added :)